AppCenter/AppCenter/Internals/Ingestion/MSHttpIngestion.h

```diff
@@ -35,10 +39,24 @@ NS_ASSUME_NONNULL_BEGIN
  * Send data to backend
  *
  * @param data A data instance that will be transformed request body.
+ * @param eTag Identity tag.
```

Comment:
nit - can you please change this to a description that doesn't mention/couple identity?

Reply:
Or if that wasn't a reference to the identity service, then can you use a different word?

Reply:
It's not "identity" it's "entity", that's on me :man_facepalming:. That's the "e" in "eTag": https://en.wikipedia.org/wiki/HTTP_ETag